text_handler.py

```diff
@@ -86,18 +88,45 @@ async def __handleMusicGrabber(self, message: types.Message):
                 await message.delete()
         except music_grabber.WrongFileSizeError as e:
             logger.error(str(e))
-            inform = await message.reply("This won't be downloaded because "
-                                         "its audio file size is greater "
-                                         "than 50M",
-                                         disable_notification=True)
-            await self.__deleteMessage(60, inform)
+            text = "This won't be downloaded because its audio file size " + \
+                "is greater than 50M"
+            inform = await message.reply(text, disable_notification=True)
+            await self.__delete_messages(60, (inform, ))
```

Comment:
same about magic numbers here. Need to make a constant for **60**
does `DELETE_SIZE_WARNING_DELAY` sound good?

```
await self.__delete_messages(DELETE_SIZE_WARNING_DELAY, (inform,))
```
looks readable to me

Reply:
came up with a little better name, see this comment https://github.com/konnovdev/cpop_bot/pull/13#discussion_r543539561

Reply:
I think it's very easy to know the mean from the context. (or from the function name if we change it) making too much constants seems does not worth the effort, we need to import them. also for other models if we just `import constants` those will be import to those functions too.

oh, we just specify the constant in text_handler.py?

Reply:
I just read about magic numbers, seems it's bad, I'll specify constants in text_handler.py